docs/src/main/sphinx/release/release-379.md

```diff
@@ -0,0 +1,87 @@
+# Release 379 (27 Apr 2022)
+
+## General
+
+* Add {doc}`/connector/mariadb`. ({issue}`10046`)
+* Improve performance of queries that contain `JOIN` and `UNION` clauses. ({issue}`11935`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+## Iceberg connector
+
+* Add support for {doc}`/sql/delete` with arbitrary predicates. ({issue}`11886`)
+## JMX connector
+
+## Kafka connector
+
+## Kinesis connector
+
+## Kudu connector
+
+## Local file connector
+
+## Memory connector
+
+## MongoDB connector
+
+## MySQL connector
+
+## Oracle connector
+
+## Phoenix connector
+
+## Pinot connector
+
+* Add support for Pinot 0.10. ({issue}`11475`)
+
+## PostgreSQL connector
+
+## Prometheus connector
+
+## Raptor connector
+
+## Redis connector
+
+## Redshift connector
+
+## SingleStore (MemSQL) connector
+
+## SQL Server connector
+
+* Apply snapshot isolation to all connections when it is enabled. ({issue}`11662`)
```

Comment:
`it` should also mention what config that is. People need to be able to read release notes and see if they are affected instead of having to hunt for information.

Reply:
"it" is snapshot isolation .. at least thats what the sentence implies .. also thats a setting on SQL Server NOT in Trino from all I know .. and there is nothing in our docs about this .. so do we need to add something?

Reply:
@mosabua The connector has undocumented `sqlserver.snapshot-isolation.disabled` config property and the default value is `false`.

Reply:
Well.. thats kinda terrible .. let me try to put a doc PR together as soon I got the release notes updated.

Reply:
https://github.com/trinodb/trino/pull/12178

Reply:
Is this a fix? Did it not work before? Or is it a new option? In that case, it should be worded as "Add support for setting snapshot isolation to SQL Server connections. This can be enabled via the xxx config property"

Reply:
Option was there before and the PR above adds docs. Not sure what happened before. It seems this is going in to support bulk insert next ..

Reply:
Previously, the property was effective only when the connection was initialized in SqlServerClient and not effective when it happened in BaseJdbcClient.